### PR TITLE
London Class 7 - Askin Ekinci - JavaScript Core 1 - Week 1

### PR DESCRIPTION
**Pull request by askinekincihf**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Review by rfloresram — COMMENTED** [review]
Very well handled the arrangement of the syntax.

File: exercises/F-strings-methods/exercise.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+const name = "Askin";
+const nameLength = name.length;
+console.log(`My name is ${name} and my name is ${nameLength} characters long.`);
```
Comment (aishak05) [inline exercises/F-strings-methods/exercise.js:4]: Well done for using this syntax to concatenate

File: exercises/K-functions-parameters/exercise2.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
 // Declare your function first
+function divide(num1, num2) {
+    const result = num1 / num2;
```
Comment (aishak05) [inline exercises/K-functions-parameters/exercise2.js:3]: you could skip this line and just have return num1/num2

File: exercises/L-functions-nested/exercise.js [hunk 3]
```diff
@@ -3,3 +3,19 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+let mentors = [mentor1, mentor2, mentor3, mentor4, mentor5];
+
+function shoutyGreeting(name) {
+    return name.toUpperCase();
+}
+
+for (let i = 0; i < mentors.length; i++) {
```
Comment (aishak05) [inline exercises/L-functions-nested/exercise.js:12]: good use of loop function

File: exercises/F-strings-methods/exercise.js (hunk 1 above)
Comment (askinekincihf) [reply exercises/F-strings-methods/exercise.js:4]: Hi Aisha, thank you very much for the feedback.

File: exercises/K-functions-parameters/exercise2.js (hunk 2 above)
Comment (askinekincihf) [reply exercises/K-functions-parameters/exercise2.js]: I amended divide function in accordance with your feedback. Thanks Aisha.

File: exercises/L-functions-nested/exercise.js (hunk 3 above)
Comment (askinekincihf) [reply exercises/L-functions-nested/exercise.js:12]: Many thanks for the feedback.

**Pull request closed without merging**